basyx/aas/adapter/http.py

```diff
@@ -143,7 +143,7 @@ def serialize(self, obj: ResponseData, stripped: bool) -> str:
                 response_elem = object_to_xml_element(obj)
                 parent.append(response_elem)
                 etree.cleanup_namespaces(parent)
-        return etree.tostring(response_elem, xml_declaration=True, encoding="utf-8")
+        return etree.tostring(response_elem, xml_declaration=True, encoding=BASE64URL_ENCODING)
```

Comment:
We shouldn't use `BASE64URL_ENCODING` here, since this is only used for encoding XML, which is independent of the base64url encoding.